README.md

```diff
@@ -49,29 +49,33 @@ If you want to contribute to **jovian**(Python package), review the [**Contribut
 <table>
     <tr>
         <th> </th>
-        <th>jovian(Python Package)</th>
+        <th>Jovian(Python Package)</th>
         <th>Jovian(Webapp)</th>
     </tr>
     <tr>
         <td>Issues or Feature Request</td>
-        <td align="center"> <a href="https://github.com/JovianML/jovian-py/issues">Forum</a> </td>
-        <td align="center"> <a href="https://github.com/JovianML/jovian-support/issues">Forum</a> </td>
+        <td align="center"> <a href="https://github.com/JovianML/jovian-py/issues" target="_blank">Forum</a> </td>
+        <td align="center"> <a href="https://jovian.ml/forum/c/help/22" target="_blank">Forum</a> </td>
```

Comment:
Seems odd, these links are still opening up in the same tab, can you verify once https://github.com/JovianML/jovian-py/tree/sid/update-readme

Reply:
Seems like Github markdown rendering doesn't support `target="_blank"`